archetypes/blog.md

```diff
@@ -1,12 +1,13 @@
 ---
 title: {{ replaceRE "[-_]" " " .Name | title }}
-linkTitle: ADD A SHORT TITLE HERE # TODO: add short title or remove this line
-date: {{ dateFormat "2006-01-02" .Date }}
+linkTitle: ADD A SHORT TITLE HERE # Mandatory, make sure that your short title. 
```

Comment:
`make sure that your short title.` is this complete? It sounds like there is more to it, eg "make sure that your short title is xyz and foo..."

just found this while trying to add a new blog post and then found this PR